src/app/layout/redirect-status/redirect-status.spec.ts

```diff
@@ -0,0 +1,48 @@
+import { NO_ERRORS_SCHEMA } from '@angular/core';
+import { async, TestBed } from '@angular/core/testing';
+import { ActivatedRoute, Router } from '@angular/router';
+import { RouterTestingModule } from '@angular/router/testing';
+
+import { RedirectStatusComponent } from './redirect-status.component';
+
+describe('Redirect Status Component', () => {
```

Comment:
This file should be named `redirect-status.component.spec.ts` - it's missing the `component` part